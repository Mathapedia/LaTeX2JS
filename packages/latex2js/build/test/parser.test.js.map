{"version":3,"file":"parser.test.js","sourceRoot":"","sources":["../../test/parser.test.ts"],"names":[],"mappings":";AAAA,kCAAgC;AAChC,IAAM,KAAK,GAAG,IAAI,eAAQ,EAAE,CAAC;AAE7B,QAAQ,CAAC,QAAQ,EAAE;IACjB,EAAE,CAAC,QAAQ,EAAE;QACX,IAAM,MAAM,GAAG,KAAK,CAAC,SAAS,EAAE,CAAC;QACjC,IAAM,MAAM,GAAG,MAAM,CAAC,KAAK,CAAC,2eAsB3B,CAAC,CAAC;QACH,MAAM,CAAC,MAAM,CAAC,CAAC,eAAe,EAAE,CAAC;IACnC,CAAC,CAAC,CAAC;IACH,EAAE,CAAC,OAAO,EAAE;QACV,IAAM,MAAM,GAAG,KAAK,CAAC,KAAK,CAAC,2eAsB1B,CAAC,CAAC;QACH,MAAM,CAAC,MAAM,CAAC,CAAC,eAAe,EAAE,CAAC;IACnC,CAAC,CAAC,CAAC;AACL,CAAC,CAAC,CAAC"}