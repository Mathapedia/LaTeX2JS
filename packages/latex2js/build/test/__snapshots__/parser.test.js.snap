// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parser parse 1`] = `
Array [
  Object {
    "lines": Array [
      "<br>",
      "Let's get to the point. The core of PSTricks is graphics!",
      "<br>",
    ],
    "settings": Object {
      "fillstyle": "none",
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "xunit": 50,
      "yunit": 50,
    },
    "type": "math",
  },
  Object {
    "env": Object {
      "h": 10,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "lines": Array [
      "\\\\psline{->}(0,-3.75)(0,3.75)",
      "\\\\psline{->}(-3.75,0)(3.75,0)",
      "\\\\pscircle(0,0){ 3 }",
    ],
    "match": Array [
      "\\\\begin{pspicture}(-5,-5)(5,5)",
      "-5",
      "-5",
      "5",
      "5",
    ],
    "plot": Object {
      "psarc": Array [],
      "psaxes": Array [],
      "pscircle": Array [
        Object {
          "data": Object {
            "cx": 250,
            "cy": 250,
            "r": 150,
          },
          "env": Object {
            "h": 10,
            "w": 10,
            "x0": -5,
            "x1": 5,
            "xunit": 50,
            "y0": -5,
            "y1": 5,
            "yunit": 50,
          },
          "fn": [Function],
          "match": Array [
            "\\\\pscircle(0,0){ 3 }",
            "0",
            "0",
            " 3 ",
          ],
        },
      ],
      "psframe": Array [],
      "psline": Array [
        Object {
          "data": Object {
            "arrows": Array [
              0,
              1,
            ],
            "dots": Array [
              0,
              0,
            ],
            "fillcolor": "black",
            "fillstyle": "solid",
            "linecolor": "black",
            "linestyle": "solid",
            "linewidth": 2,
            "x1": 250,
            "x2": 250,
            "y1": 437.5,
            "y2": 62.5,
          },
          "env": Object {
            "h": 10,
            "w": 10,
            "x0": -5,
            "x1": 5,
            "xunit": 50,
            "y0": -5,
            "y1": 5,
            "yunit": 50,
          },
          "fn": [Function],
          "match": Array [
            "\\\\psline{->}(0,-3.75)(0,3.75)",
            undefined,
            "{->}",
            "0",
            "-3.75",
            "(0,3.75)",
            "0",
            "3.75",
          ],
        },
        Object {
          "data": Object {
            "arrows": Array [
              0,
              1,
            ],
            "dots": Array [
              0,
              0,
            ],
            "fillcolor": "black",
            "fillstyle": "solid",
            "linecolor": "black",
            "linestyle": "solid",
            "linewidth": 2,
            "x1": 62.5,
            "x2": 437.5,
            "y1": 250,
            "y2": 250,
          },
          "env": Object {
            "h": 10,
            "w": 10,
            "x0": -5,
            "x1": 5,
            "xunit": 50,
            "y0": -5,
            "y1": 5,
            "yunit": 50,
          },
          "fn": [Function],
          "match": Array [
            "\\\\psline{->}(-3.75,0)(3.75,0)",
            undefined,
            "{->}",
            "-3.75",
            "0",
            "(3.75,0)",
            "3.75",
            "0",
          ],
        },
      ],
      "pspicture": Array [],
      "psplot": Array [],
      "pspolygon": Array [],
      "psset": Array [],
      "rput": Array [],
      "slider": Array [],
      "userline": Array [],
      "uservariable": Array [],
    },
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "pspicture",
  },
  Object {
    "lines": Array [
      "<br>",
      "which can be produced using the following $\\\\TeX$:",
      "<br>",
    ],
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "math",
  },
  Object {
    "lines": Array [
      "\\\\begin{pspicture}(-5,-5)(5,5)",
      "\\\\psline{->}(0,-3.75)(0,3.75)",
      "\\\\psline{->}(-3.75,0)(3.75,0)",
      "\\\\pscircle(0,0){ 3 }",
      "\\\\end{pspicture}",
    ],
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "verbatim",
  },
  Object {
    "lines": Array [
      "<br>",
    ],
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "math",
  },
]
`;

exports[`Parser parser 1`] = `
Array [
  Object {
    "lines": Array [
      "<br>",
      "Let's get to the point. The core of PSTricks is graphics!",
      "<br>",
    ],
    "settings": Object {
      "fillstyle": "none",
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "xunit": 50,
      "yunit": 50,
    },
    "type": "math",
  },
  Object {
    "env": Object {
      "h": 10,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "lines": Array [
      "\\\\psline{->}(0,-3.75)(0,3.75)",
      "\\\\psline{->}(-3.75,0)(3.75,0)",
      "\\\\pscircle(0,0){ 3 }",
    ],
    "match": Array [
      "\\\\begin{pspicture}(-5,-5)(5,5)",
      "-5",
      "-5",
      "5",
      "5",
    ],
    "plot": Object {
      "psarc": Array [],
      "psaxes": Array [],
      "pscircle": Array [
        Object {
          "data": Object {
            "cx": 250,
            "cy": 250,
            "r": 150,
          },
          "env": Object {
            "h": 10,
            "w": 10,
            "x0": -5,
            "x1": 5,
            "xunit": 50,
            "y0": -5,
            "y1": 5,
            "yunit": 50,
          },
          "fn": [Function],
          "match": Array [
            "\\\\pscircle(0,0){ 3 }",
            "0",
            "0",
            " 3 ",
          ],
        },
      ],
      "psframe": Array [],
      "psline": Array [
        Object {
          "data": Object {
            "arrows": Array [
              0,
              1,
            ],
            "dots": Array [
              0,
              0,
            ],
            "fillcolor": "black",
            "fillstyle": "solid",
            "linecolor": "black",
            "linestyle": "solid",
            "linewidth": 2,
            "x1": 250,
            "x2": 250,
            "y1": 437.5,
            "y2": 62.5,
          },
          "env": Object {
            "h": 10,
            "w": 10,
            "x0": -5,
            "x1": 5,
            "xunit": 50,
            "y0": -5,
            "y1": 5,
            "yunit": 50,
          },
          "fn": [Function],
          "match": Array [
            "\\\\psline{->}(0,-3.75)(0,3.75)",
            undefined,
            "{->}",
            "0",
            "-3.75",
            "(0,3.75)",
            "0",
            "3.75",
          ],
        },
        Object {
          "data": Object {
            "arrows": Array [
              0,
              1,
            ],
            "dots": Array [
              0,
              0,
            ],
            "fillcolor": "black",
            "fillstyle": "solid",
            "linecolor": "black",
            "linestyle": "solid",
            "linewidth": 2,
            "x1": 62.5,
            "x2": 437.5,
            "y1": 250,
            "y2": 250,
          },
          "env": Object {
            "h": 10,
            "w": 10,
            "x0": -5,
            "x1": 5,
            "xunit": 50,
            "y0": -5,
            "y1": 5,
            "yunit": 50,
          },
          "fn": [Function],
          "match": Array [
            "\\\\psline{->}(-3.75,0)(3.75,0)",
            undefined,
            "{->}",
            "-3.75",
            "0",
            "(3.75,0)",
            "3.75",
            "0",
          ],
        },
      ],
      "pspicture": Array [],
      "psplot": Array [],
      "pspolygon": Array [],
      "psset": Array [],
      "rput": Array [],
      "slider": Array [],
      "userline": Array [],
      "uservariable": Array [],
    },
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "pspicture",
  },
  Object {
    "lines": Array [
      "<br>",
      "which can be produced using the following $\\\\TeX$:",
      "<br>",
    ],
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "math",
  },
  Object {
    "lines": Array [
      "\\\\begin{pspicture}(-5,-5)(5,5)",
      "\\\\psline{->}(0,-3.75)(0,3.75)",
      "\\\\psline{->}(-3.75,0)(3.75,0)",
      "\\\\pscircle(0,0){ 3 }",
      "\\\\end{pspicture}",
    ],
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "verbatim",
  },
  Object {
    "lines": Array [
      "<br>",
    ],
    "settings": Object {
      "fillstyle": "none",
      "h": 10,
      "linecolor": "black",
      "linestyle": "solid",
      "runit": 50,
      "unit": 50,
      "w": 10,
      "x0": -5,
      "x1": 5,
      "xunit": 50,
      "y0": -5,
      "y1": 5,
      "yunit": 50,
    },
    "type": "math",
  },
]
`;
